src/probe.c

```diff
@@ -227,20 +218,10 @@ int main(int argc, char **argv)
 
         telem_log(LOG_INFO, "Listening on socket...\n");
 
-        /* Calculate the spool directory size */
-        if (is_spool_valid()) {
-                daemon.is_spool_valid = true;
-                daemon.current_spool_size = get_spool_dir_size();
-        }
-
+        bool daemon_recycling_enabled = daemon_recycling_enabled_config();
         int spool_process_time = spool_process_time_config();
-
-        spool_records_loop(&(daemon.current_spool_size));
-        time_t last_spool_run_time = time(NULL);
         time_t last_daemon_start_time = time(NULL);
 
-        bool daemon_recycling_enabled = daemon_recycling_enabled_config();
-
         ret = update_machine_id();
```

Comment:
Looks like telemprobd is managing the machine ID then.

Perhaps a new service can be created to manage it instead? With the split of telemd, the machine ID no longer makes sense for either of the new daemons to deal with (IMO).